File: app/src/main/java/app/vodio/com/vodio/database/retrofit/services/UserService.java
package app.vodio.com.vodio.database.retrofit.services;

import app.vodio.com.vodio.beans.User;
import app.vodio.com.vodio.database.DatabaseResponse;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface UserService {

    @GET("user/get.php")
    Call<User> getUser(@Query("login") String login, @Query("password") String password);

    @GET("user/create.php")
    Call<DatabaseResponse> createUser(@Query("login") String login, @Query("password") String password,
                                      @Query("name") String name, @Query("surname") String surname,@Query("email") String email);
}

File: app/src/main/java/app/vodio/com/vodio/database/retrofit/services/UserService.kt
package app.vodio.com.vodio.database.retrofit.services;


import app.vodio.com.vodio.beans.User;
import app.vodio.com.vodio.database.DatabaseResponse;
import io.reactivex.Observable
import io.reactivex.Single
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import java.util.*

interface UserService {

    @GET("user/get.php")
    fun getUser_old(@Query("login") login: String, @Query("password") password: String): Call<User>

    @GET("user/get.php")
    fun getUser(@Query("login") login: String, @Query("password") password: String): Single<User>

    @GET("user/create.php")
    fun createUser_old(@Query("login") login: String, @Query("password") password: String,
                   @Query("name") name: String, @Query("surname") surname: String, @Query("email") email: String): Call<DatabaseResponse>

    @GET("user/create.php")
    fun createUser(@Query("login") login: String, @Query("password") password: String,
                   @Query("name") name: String, @Query("surname") surname: String, @Query("email") email: String): Single<DatabaseResponse>
}